dev-docs/show-video-with-a-dfp-video-tag.md

```diff
@@ -65,126 +58,85 @@ Don't forget to add your own valid placement ID.
 
 ```javascript
 var videoAdUnit = {
-  code: 'video',
-  sizes: [640,480],
-  mediaType: 'video',
-  bids: [
-    {
-      bidder: 'appnexusAst',
-      params: {
-        placementId: '123456'  // <-- Replace this!
+    code: 'video',
+    sizes: [640, 480],
+    mediaType: 'video',
+    bids: [{
+        bidder: 'appnexusAst',
+        params: {
+            placementId: '9333431',
             video: {
-              skippable: true
+                skippable: true,
+                playback_methods: ['auto_play_sound_off']
             }
-      }
-    }
-  ]
+        }
+    }]
 };
 ```
 
-### 2. Implement Custom Price Buckets to work around the default CPM cap of $20
-
-By default, Prebid.js caps all CPMs at $20.  As a video seller, you may expect to see CPMs over $20.  In order to receive those bids, you'll need to implement custom price buckets using the [`setPriceGranularity`]({{site.github.url}}/dev-docs/publisher-api-reference.html#customCPMObject) method.
+### 2. Implement Custom Price Buckets
 
-For instructions, see [Custom Price Bucket with `setPriceGranularity`]({{site.github.url}}/dev-docs/examples/custom-price-bucket-using-setpricegranularity.html).
+By default, Prebid.js caps all CPMs at $20.  As a video seller, you may expect to see CPMs over $20.  In order to receive those bids, you'll need to implement custom price buckets using the [`setPriceGranularity`]({{site.baseurl}}/dev-docs/publisher-api-reference.html#customCPMObject) method.
 
-### 3. Request bids, build a video tag, and invoke the player
+For instructions, see [Custom Price Bucket with `setPriceGranularity`]({{site.baseurl}}/dev-docs/examples/custom-price-bucket-using-setpricegranularity.html).
 
-Next, do the standard Prebid "add ad units and request bids" dance.
-In the example below, we've added some code that is not strictly
-necessary, but was helpful during development.  Specifically, we log:
+### 3. Request bids, build video URL, invoke player
 
-+ the `bids` object we got back from our demand sources
-+ a notice telling us whether there was video demand
-+ the URL of the VAST creative, if any (this is also helpful if you
-  don't necessarily use DFP)
+Next, we need to do the standard Prebid "add ad units and request bids" dance.  In the example below, our callback builds the video URL the player needs using the `buildVideoUrl` method from the DFP ad server module that we built into our copy of Prebid.js in the **Prerequisites** section.
 
-To optimize setup for low latency, we recommend that this code (and that referenced above in step #1) be added to the page header.
+This method wraps the video URL in VAST tags and tries to add it to a server-side Prebid Cache. The use of the cache is transparent to the user and doesn't require a login or account of any kind.  It's designed to simplify ad ops workflows by enabling all Prebid video demand partners to serve through a single common video URL.
```

Comment:
This isn't technically accurate. The video was added to the server-side cache as soon as the bidder responded with it--before the bidsBackHandler even executes. This function just builds a DFP URL which plays nicely with the AdOps setup we recommend.

Also... I'll defer to your expertise here, but I'm not really sure why the cache is important here. I've always thought of this page as a "how do I do it?" tutorial. The cache is valuable for someone asking "_should_ I do it?" I do think we should be transparent about it, because we'll need to work on scaling prebid-cache as the load increases, and there's some risk to impression loss if (& when, for beta adopters) it goes down. But... that strikes me as a different audience from the person who just needs to set up the page to use it?

Lastly: it may be worth noting here that `params` supports anything from [the DFP API](https://support.google.com/dfp_premium/answer/1068325?hl=en). Though, [the docs in our code](https://github.com/prebid/Prebid.js/blob/master/modules/dfpAdServerVideo.js#L10) also say that... so maybe it's not that big of a deal. If I were looking at this page in a vacuum, though, I'd be very confused by the random `iu` parameter.